### PR TITLE
[docs] update readme

### PR DESCRIPTION
**Pull request by adsteel**
How'm I doing?

@mhuggins @brandonc

### CODE REVIEW

File: lib/clone_kit/rules/except.rb [hunk 1]
```diff
@@ -2,20 +2,24 @@
 
 module CloneKit
   module Rules
+    #
+    # Removes attributes defined by an array
+    # of keys
```
Comment (mhuggins) [inline lib/clone_kit/rules/except.rb]: :neckbeard: The line breaks seem arbitrary.

File: lib/clone_kit/rules/safe_remap.rb [hunk 2]
```diff
@@ -4,6 +4,10 @@
 
 module CloneKit
   module Rules
+    #
+    # Operates like Remap, but returns a default value instead of `nil`.
+    # When the default is used, a warning is added to the operation.
```
Comment (mhuggins) [inline lib/clone_kit/rules/safe_remap.rb]: It's also treated as a warning instead of an error in the event outlet.

**Review by mhuggins — APPROVED** [review]

File: lib/clone_kit/rules/except.rb (hunk 1 above)
Comment (adsteel) [reply lib/clone_kit/rules/except.rb]: I don't have an auto-formatter for comments, so they kind of are.

Comment (adsteel) [reply lib/clone_kit/rules/except.rb]: I'll clean this one up though. Seems a little short.

**Pull request merged**